layout/navigation.html

```diff
@@ -18,11 +18,97 @@
   <h1>Primary Navigation<span class="status dev-ready">Complete</span></h1>
   <p>The primary navigation is used for the main sections of the application or for dropdown categories that organize sections into higher level groups.</p>
 
+  <div class="row" id="side-nav">
+    <div class="col-md-4">
+      <h2>Left Navigation</h2>
+      <p>The Left Navigation component is used on all web applications that desire to implement a highly compliant version of the Helix Design Language.</p>
+      <ul>
+        <li>Can extend to fit any navigation IA via scrolling when the list of navigational items exceeds the screen view-height.</li>
+        <li>Preferred implementation includes only primary and secondary navigational links.</li>
+          <ul>Note: A less preferred implementation that includes <a href="{{page.url}}#tertiary"> tertiary navigational</a> elements is also provided.</ul>
+      </ul>
+    </div>
+    <div class="col-md-7 col-md-offset-1">
+      <figure>
+        <div class="image bg-light border">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left.svg">
+        </div>
+      </figure>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-md-4">
+      <h2>Left Navigation Color and Typeface</h2>
+      <ul>
+        <li>Font: Roboto Regular 16px / 1em</li>
+        <li>Color: Gray 400 #D8D8D8</li>
+        <li>Text Transform: Uppercase</li>
+      </ul>
+    </div>
+    <div class="col-md-7 col-md-offset-1">
+      <figure>
+        <div class="image bg-light border">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left-color-type.svg">
+        </div>
+      </figure>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-md-4">
+      <h2>Left Navigation Spacing</h2>
+      <ul>
+        <li>First primary navigational element begins 60px from top of containing div.</li>
+      </ul>
+    </div>
+    <div class="col-md-7 col-md-offset-1">
+      <figure>
+        <div class="image bg-light border">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left-spacing-horizontal.svg">
+        </div>
+        <figcaption>Containing div spacing specs</figcaption>
+      </figure>
+    </div>
+    <div class="col-md-4">
+      <ul>
+        <li>16px / 1em top and bottom padding for navigational elements</li>
+      </ul>
+    </div>
+    <div class="col-md-7 col-md-offset-1">
+      <figure>
+        <div class="image bg-light border">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left-spacing-vertical.svg">
+        </div>
+        <figcaption>Vertical spacing between navigational items</figcaption>
+      </figure>
+    </div>
+    <div class="col-md-4">
```

Comment:
Same here; seems to be missing the end of one row and the beginning of the next.